data/35e/wizards_of_the_coast/supplement/expanded_psionics_handbook/xph_abilities.lst

```diff
@@ -179,7 +179,9 @@ Psionic Talent Power Points	CATEGORY:Special Ability	TYPE:ClassAbility.Internal
+Powers Known			KEY:Psion ~ Powers Known			CATEGORY:Special Ability	TYPE:Class Feature.Psion Class Feature.SpecialQuality	DESC:A psion begins play knowing three psion powers of your choice. Each time he achieves a new level, he unlocks the knowledge of new powers. Choose the powers known from the psion power list, or from the list of powers of your chosen discipline. You cannot choose powers from disciplines other than your chosen discipline. (Exception: The feats Expanded Knowledge, page 46, and Epic Expanded Knowledge, page 34, do allow a psion to learn powers from the lists of other disciplines or even other classes.) A psion can manifest any power that has a power point cost equal to or lower than his manifester level. The number of times a psion can manifest powers in a day is limited only by his daily power points. A psion simply knows his powers; they are ingrained in his mind. He does not need to prepare them (in the way that some spellcasters prepare their spells), though he must get a good night's sleep each day to regain all his spent power points. The Difficulty Class for saving throws against psion powers is 10 + the power's level + the psion's Intelligence modifier.
+#For example, a 9th-level psion (with a total of 72 power points) could manifest a power costing 1 power point seventy-two times in one day, a power costing 9 power points eight times in one day, or any other combination of manifestations that does not exceed 72 power points in total.
```

Comment:
Why are we interposing commented out text here and removing the SOURCEPAGE tag?

Reply:
clutter removed - I'd cut the end of line off (as it's an 'example', and not appropriate for the character sheet output, rather than deleting it completely, I'd just commented it out, and had overlooked the fact that the sourcepage tag was at the very end. Latest commit should resolve that, and the remaining commented code bits